Cargo.toml

```diff
@@ -16,7 +16,7 @@ homepage = "https://amethyst.rs/"
 repository = "https://github.com/amethyst/amethyst"
 
 readme = "README.md"
-license = "MIT/Apache-2.0"
+license = "MIT OR Apache-2.0"
```

Comment:
@kirusfg
This has to be rejected, we can't have 3rd party changes to the license
Even tho it's just formating.

Reply:
This change was committed on Aug 15 and not by me.

Also, I barely use git and GitHub, so I need to ask whether I did this right. Whenever the main branch is updated, how does one get all those updates merged into the branch they are working on? I did a rebase off of the main branch, it added all those commits (one of which introduced that change to the license you mentioned) and duplicated my work onto the head of the branch. Is this the correct way to do it?

Reply:
Oh, see if you can squash / update it

Github itself should have buttons for it if you go into your branch